js/controller/NavigationController.js

```diff
@@ -265,6 +265,10 @@ function ($route, FEED_TYPE, FeedResource, FolderResource, ItemResource,
         }
     };
 
+    this.openFeedUrl = function (feed) {
+        window.open(feed.url, '_blank');
```

Comment:
Needs a rel="noopener" or vulnerable to attacks

Reply:
oh, nice catch, thanks! I updated the pull request to use a <a> tag with rel="noopener". I also added noreferrer to keep the nextcloud server url private.